### DISCUSSION TITLE
Construct equivalent for parse_obj_as?

### DISCUSSION THREAD

**Discussion by jasonhuling** [pydantic/pydantic#6203] [Question]
Is there an equivalent to the `construct` method but for scenarios where you may be using `parse_raw_as` or `parse_obj_as`?

I'm currently maintaining a library that interacts with a few different REST APIs. The pydantic models are auto-generated from the APIs' swagger documentation, which is not always 100% accurate. Typically optional fields are not always called out correctly, or constraints may be incorrect. I can't always catch these ahead of time, and would like to provide an option to skip validation so that the user can still utilize the library without being blocked by bad data. I'm also considering just always skipping validation.

Right now I use either `parse_raw_as` or `parse_obj_as` to parse the response body. I saw that https://github.com/pydantic/pydantic/issues/897 provides the `construct` method, but as a method on a `BaseModel`, so not something that I can swap in directly for my current scenario. So just looking to see if I may be missing some other option.

**Comment by adriangb** [comment]
It sounds like the validation isn't providing much value to you. Given that's the case, what value is Pydantic providing you? That might help inform the best solution here.
